distsql/request_builder.go

```diff
@@ -45,6 +45,8 @@ type RequestBuilder struct {
 	kv.Request
 	is  infoschema.InfoSchema
 	err error
+	// When SetDAGRequest is called, builder will also this field.
```

Comment:
What does "builder will also this field" mean?

Reply:
Typo ... 'will also set this field'